src/renderer/services/midgard/service.ts

```diff
@@ -269,6 +270,11 @@ const networkInfo$: Rx.Observable<NetworkInfoRD> = reloadNetworkInfo$.pipe(
   shareReplay()
 )
 
+const apiEndpoint$: Rx.Observable<O.Option<string>> = byzantine$.pipe(
```

Comment:
maybe we should start using `Either` monad ? it's gonna be helpful to understand what was wrong while debugging and detecting a problem